.plzconfig

```diff
@@ -130,6 +130,10 @@ Type = bool
 Inherit = true
 Help = If set, the licences field on pip_library and python_wheel will be mandatory
 
+[PluginConfig "verbosity"]
+DefaultVault = "info"
+Help = Passes verbosity level to wheel_tool, for example: wheel_tool --verbosity info
+Optional = true
```

Comment:
Not sure why I get 

```
Error building target //third_party/python3:_PyInquirer#download: exit status 2
Error: Option '--verbosity' requires an argument.
```

When `verbosity` isn't set in `.plzconfig` given that `Optional = true`

Reply:
Because you're still passing --verbosity even if verbosity ins't set? I think you want `optional=false` but have a default option. Optional just means that it can be unset i.e. you can do `Verbosity =`.